content/en/blog/2021-03-19-a-parrot-trainer-eats-crow.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "A Parrot Trainer Eats Crow"
+slug: "a-parrot-trainer-eats-crow"
+date: "2021-03-19T08:42:21-04:00"
+draft: false
+image_webp: images/blog/parrots.webp
+image: images/blog/parrots.jpg
+author: Rebecca Bilbro
+description : "Models trained on massive datasets using millions of parameters can both have low bias and also be very biased - what can we in the ML community do about this?"
+---
+
+In this post, we'll consider how it is that models trained on massive datasets using millions of parameters can be both "low bias" and also *very biased*, and begin to think through what we in the ML community might be able to do about it.
+
+## Birds on a Wire
```

Comment:
I really like how you stayed on the bird theme throughout the entire post - it makes the post more engaging but also unifies it as a well-collected whole!